src/v4/core/computedvalue.ts

```diff
@@ -107,7 +107,7 @@ export class ComputedValue<T> implements IObservable, IComputedValue<T>, IDeriva
      * This is useful for working with vectors, mouse coordinates etc.
      */
     constructor(options: IComputedValueOptions<T>) {
-        invariant(options.get, "missing option for computed: get")
+        invariant(options.get, "missing option for computed: get " + options.name)
```

Comment:
It might be better this way. Also two lines down there is a default value for a name if not specified, it should be used, so probably best to move that assignment above this invariant.

```suggestion
        invariant(options.get, `missing option for computed ${options.name}: get`)
```

Reply:
Yes, template literals are the way to go however, I didn't use them because I wanted to follow the current style of writing invariant messages.